### PR TITLE
Adds name amalia.

### PR DESCRIPTION
**Pull request by kaxcode**

### CODE REVIEW

File: collection/amalia.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "amalia",
+    "meaning": "Work",
+    "translations": {
+        "fra": "émeline",
+        "pol": "amelia",
+        "por": "amália",
+        "ita": "amelia",
+        "lit": "Amalija"
```
Comment (arminkhoshbin) [inline collection/amalia.json]: Can you please update this translation to use lowercase letters?

**Comment by arminkhoshbin** [comment]
Hi @kaxcode,
Thanks for your contribution. Can you please address the requested changes?
Tag me here when it's done and I'll happily merge this for you :)

**Comment by kaxcode** [comment]
Changes made.

**Comment by arminkhoshbin** [comment]
Fantastic! Thanks @kaxcode.
Merged #1

**Pull request merged**